min_version_test/lib/main.dart

```diff
@@ -112,9 +112,13 @@ class _MyHomePageState extends State<MyHomePage> {
       final log = Logger('_MyHomePageState');
 
       try {
+
+        // TODO Handle web file error
+
         final file = File('response.txt');
         final response = await dio.get<String>('https://flutter.dev/');
-        await file.writeAsString(response.data ?? 'no response');
+        print(response);
+        // await file.writeAsString(response.data ?? 'no response');
```

Comment:
Why did we change this? what is the error to be handled?

Reply:
Can we fix the sample instead using conditional imports?

Example:
```dart
import '_io_environment_variables.dart'
    if (dart.library.html) '_web_environment_variables.dart'
```

Where the code does the same but one of them is NoOp, we can use this approach to conditionally import what we need.

Reply:
Split the transaction sample in two files. One for web, the other for iOS/Android.